### ISSUE TITLE
Pause gifs

### ISSUE THREAD

**Issue by forteller** [wireapp/wire-android#744]
I love gifs as much as the next meme loving semi-geek. But I am also a bit sensitive to lots of movements in my periphery vision. I'd very much appreciate it if taping (clicking on desktop/web) on a gif would pause/unpause it.

On Android the menu that now shows up on tapping could be moved to long press instead.

Thanks!

**Comment by darkbluelight** [comment]
I completely agree with @forteller I'd very much appreciate it too! At the moment I delete gifs after some 15 seconds or so because the movement gets annoying.

**Comment by forteller** [comment]
Actually, when I think about it, gifs should automatically pause after some time (say 10 seconds), and then just start again if you tap/click on them.

**Comment by forteller** [comment]
Just want to know if anyone has looked at this in any way since I reported it. Thanks!

**Comment by selurvedu** [comment]
I have an exactly opposite problem – the gifs are not looped, i.e., they are paused after being played back only once. :)
